### PR TITLE
PLAD-53 feature completed

### PR DESCRIPTION
**Pull request by AliceCasali**

### CODE REVIEW

File: frontend/lib/models/placement.dart [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+      institutions: json["institutions"]
+          .map((institution) => Institution.fromJson(institution))
+          .toList(),
+      majors: json["majors"].map((major) => Major.fromJson(major)).toList(),
+      // ! This is throwing the following exception: 
+      // ! Expected a value of type 'Map<String, dynamic>', but got one of type 'MappedListIterable<dynamic, dynamic>'
+      // ! Fix this when needed
+      // skills: json["skills"].map((entry) {
+      //   entry.value = entry.value.map((skill) => Skill.fromJson(skill)).toList();
+      // }),
```
Comment (aidadenisa) [inline frontend/lib/models/placement.dart:69]: Why did we comment this? Do we need it?

File: backend/DAO/placementDAO.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+            .insert({
+                position: details.position,
+                working_hours: workingHs,
+                start_period: details.startPeriod,
+                end_period: details.endPeriod, 
+                salary: salary,
+                description_role: details.descriptionRole,
+                institution: details.institution, 
+                major: details.major
+            }, ['id', 'position', 'working_hours', 'start_period', 'end_period', 'salary', 'description_role', 'institution', 'major']);
```
[the remark this answers is not in the document]
Comment (Fabs97) [reply backend/DAO/placementDAO.js]: Institutions and majors are passed within the placement object with "institution**s**" and "major**s**" naming

File: frontend/lib/models/placement.dart (hunk 1 above)
Comment (Fabs97) [reply frontend/lib/models/placement.dart:69]: At the moment being, we do not need it. This is the returning object from the database handling and, at the moment, the required skills are not showed, therefore these lines can be handled with fresh eyes in another story/sprint

File: backend/DAO/placementDAO.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (aidadenisa) [reply backend/DAO/placementDAO.js]: @Fabs97 this is outdated, it was the one from the commit before the changes to the backend.

**Review by aidadenisa — APPROVED** [review]

**Pull request merged**